tokio-postgres/tests/test/main.rs

```diff
@@ -897,6 +897,44 @@ async fn query_one() {
         .unwrap();
 }
 
+#[tokio::test]
+async fn query_typed_one() {
```

Comment:
I would like this test to exercise a bit more:
- Pass multiple params for different types (like query_typed_no_transaction, which passes a TEXT and INT) type
- Assert return values

While currently, the implementation which uses the same machinery as query_typed, better tests will ensure that the behavior remains unchanged even if that is not the case.